docs/Deploying.md

```diff
@@ -21,108 +9,55 @@ Use it as a template to create your own:
-
-## Set environment variables for Tomcat
-
-The `PORTAL_HOME` environment variable needs to be available to
-the `cbioportal.war` file which runs within the Tomcat server,
-so that it can find the configuration file. To make it available to Tomcat,
-edit your Tomcat startup file (typically `$CATALINA_HOME/bin/setenv.sh`)
-and add a line like the following, pointing it to the folder containing
-`portal.properties`:
+For more information about the `portal.properties` file, see the
+[reference](portal.properties-Reference.md) page.
 
-    export PORTAL_HOME=/Users/johndoe/cbioportal
+Several scripts of cBioPortal use this `portal.properties` file to get info
+like db connection parameters. You can indicate the folder where this file is
+with an environment variable:
 
-In additon, add a line to make Tomcat pass `-Dauthenticate=false`
-as a JVM argument, or replace the word ‘false’ by the method to use:
-
-    CATALINA_OPTS='-Dauthenticate=false'
-
-## Add the MySQL JDBC Driver to Apache Tomcat
-
-A proper JDBC driver will also need to be accessible by Apache Tomcat.  If using MySQL, the [Connector/J](http://dev.mysql.com/downloads/connector/j/) driver jar file should be placed in `$CATALINA_HOME/lib`.
-
-More information on configuring Apache Tomcat connection pooling can be found [here](http://tomcat.apache.org/tomcat-7.0-doc/jndi-datasource-examples-howto.html).
-
-***We have reports that the Tomcat package that comes with (at least) Ubuntu 14.04 cannot handle the connection pool from resources.  If you are encountering this is, we suggest you [download the Tomcat archive from Apache and install from there](https://www.digitalocean.com/community/tutorials/how-to-install-apache-tomcat-8-on-ubuntu-14-04).***
+```
+export PORTAL_HOME=$HOME/cbioportal
+```
 
-## Set up the Database Connection Pool
+if your properties file is at `PORTAL_HOME/portal.properties`
 
-Apache Tomcat provides the database database connection pool to the cBioPortal.  To setup a database connection pool managed by Tomcat, add the following line to `$CATALINA_HOME/conf/context.xml`, making sure that the properties match your system:
+## Run the app
 
-``` xml
-<Context>
-    ...
-    <Resource name="jdbc/cbioportal" auth="Container"
-    type="javax.sql.DataSource"
-    maxActive="100" maxIdle="30" maxWait="10000"
-    username="cbio_user" password="somepassword"
-    driverClassName="com.mysql.jdbc.Driver"
-    connectionProperties="zeroDateTimeBehavior=convertToNull;"
-    testOnBorrow="true"
-    validationQuery="SELECT 1"
-    url="jdbc:mysql://localhost:3306/cbioportal"/>
-...
-</Context>
 ```
-## Deploy the cBioPortal WAR
-
-A tomcat server is usually started by running the following command:
-
-``` bash
-$CATALINA_HOME/bin/catalina.sh start
+java \
+    -Dauthorization=false \
+    -Dauthenticate=false \
+    -Ddbconnector=dbcp \
+    -jar \
+    portal/target/dependency/webapp-runner.jar \
+    --path /cbioportal \
+    portal/target/cbioportal.war
 ```
```

Comment:
can we still configure 
1. the maximum number of active connections that can be allocated from this pool
2. the maximum number of connections that can remain idle in the pool, without extra ones being released,
since default values are set very small?

Reply:
It's all set here for the `dbcp` connection: https://github.com/cBioPortal/cbioportal/blob/fbd69216311e484947f55e30b180ba7244b6d824/core/src/main/java/org/mskcc/cbio/portal/dao/JdbcDataSource.java#L26-L36. We currently don't provide a way to change these, but we set them to what we were using in production

Reply:
a good job, all configured already!!

Reply:
if we can add some new properties in portal.properties to allow users to adjust those numbers that would be really helpful.

Reply:
I'm curious - did you play around with these values on your installation @khzhu or @fedde-s ? I imagine most people copy+pasted what we had in the docs. Feels a bit silly to introduce config that nobody will use

Reply:
I do, when people complains the portal is too slow or take a long time to get response. As we move to clouds, with those advance performance monitor tools those numbers can be fine tuned or explored :) .

Reply:
@khzhu do you have any insights already to share on this? We haven't touched much of the mysql configuration on our end

Reply:
it is a task we've given to Airgate, a professional cloud service firm Azure hired, to help come up with a configuration that best leverages the AKS we have. We will share our experience once it is done.

Reply:
i created a follow up issue for this https://github.com/cBioPortal/cbioportal/issues/6118